### DISCUSSION TITLE
Regarding "cnd integration proposal"

### DISCUSSION THREAD

**Discussion by vinothv84** [apache/netbeans#5882] [Q&A]
Dear Community,
I am a long-time user of NetBeans C++. IMHO, NetBeans C++ is superior to eclipse or other IDEs in-terms of remote development (especially), etc. Right now, we can use 8.2 portal C++ plugin as a workaround, which still works thankfully as of NB17.

As a pure user, I have followed this proposal (below link) in the confluence page for some time, in a hope that it will be released soon.

https://cwiki.apache.org/confluence/display/NETBEANS/cnd+integration+proposal%3A+cnd-small

(If I may) I would like to ask if this plan/proposal is still valid and part of the roadmap, or it is kind of abandoned because of some reason. I sincerely hope that one day it is going to come to life again.
Would appreciate any response or update in this regard.

Thanks in advance.

**Comment by PetrMiksik** [comment]
CND lives again :-)
https://github.com/piotrhoppe/netbeans-cnd

**Reply by vinothv84** [reply]
I tried to build this, but i am getting some errors for cnd-lsp. Some yenta lib 1.2-SNAPSHOT missing.. this is not a maven artifact, not available in maven central.. so i am not sure from where did he obtain this..
Also it would be better if this can be released as plugin in the plugin portal?

**Reply by PetrMiksik** [reply]
It's probably library allowing retroactive hack of NBM modules. I've found it on GitHub here:
https://github.com/jglick/yenta

**Reply by PetrMiksik** [reply]
Did you use the installation script provided by the author? The nbm packages contains additional libraries, which need to be extracted to the NetBeans folder. I see the release artifact with the compiled nbm modules does not contain it.

You can download nbm packages with installation scripts from the link below:
https://file.inova.cz/sharing/yAnArAu3g

It contains 3 folder:
netbeans-cnd-0.1.0.1-nbms - contains module files with installation script
netbeans-cnd-0.1.0.1-update-center - definition file to NetBeans update center
scripts - patch of dlightmodule to add cnd.lsp as a friend module to enable access to NativeExecution
